ui/app/components/app/permissions-connect-header/index.scss

```diff
@@ -38,7 +38,14 @@
     color: $Grey-500;
   }
 
+  &__text {
+    width: 100%;
```

Comment:
The direction here will be incorrectly reversed on right-to-left domains - we can stop this from happening with a `rtl:ignore` directive:
```suggestion
    /*rtl:ignore*/
    width: 100%;
```

Reply:
TIL! 05f088c